src/Http/Controllers/VoyagerBreadController.php

```diff
@@ -160,16 +160,25 @@ public function update(Request $request, $id)
         // Check permission
         Voyager::canOrFail('edit_'.$dataType->name);
 
-        $data = call_user_func([$dataType->model_name, 'findOrFail'], $id);
+        //Validate fields with ajax
+        $val = $this->validateBread($request->all(), $dataType->addRows);
```

Comment:
Heads up there is a bug here still not corrected on master branch.
We should validate `$dataType->editRows`
Fixed in PR https://github.com/the-control-group/voyager/pull/1198